rpc/README.md

```diff
@@ -0,0 +1,26 @@
+# tendermint-rpc
+
+A rust implementation of the core types returned by a Tendermint node's RPC 
+endpoint. 
+These can be used to deserialize JSONRPC responses.
+All networking related features will be feature guarded to keep the dependencies small 
+in cases where only the core types are needed.
+
+Releases of this crate should be compatible to the same [tendermint] version as
+the [tendermint.rs] crate with the same version number.    
```

Comment:
```suggestion
the [tendermint-rs] crate with the same version number.    
```

Is this a typo? Also we should get tendermint.rs !!!

Reply:
![image](https://user-images.githubusercontent.com/7441766/85131396-fcf34180-b236-11ea-9dc0-b211ea24c4e5.png)

Reply:
> Also we should get tendermint.rs !!!

@ebuchman